### ISSUE TITLE
Double IDs showing

### ISSUE THREAD

**Issue by fsktom** [mezz/JustEnoughItems#1252]
Hello. By hovering over an item in JEI I see the item ID (#4590/0) right next to the name, under it the ID (extrautils2:machine), sometimes under it I see "NBT: 1 tag(s)", under it the mod name, and under it the ID (extrautils2:machine).
[Img 1](https://i.imgur.com/yKWZtMR.png) - [Img 2](https://i.imgur.com/zGnuqA8.png) - [Img 3](https://i.imgur.com/3LXgIe7.png)
I've tried disabling NEI, Hwyla, and Wawla but nothing helped.
Help would be appreciated.

**Comment by MauveCloud** [comment]
Do you have EnderCore installed?  It has an option "showRegistryNameTooltips" that might be where one of the "extrautils2:machine" entries is coming from.

**Comment by fsktom** [comment]
Yeah, I have EnderCore installed. I'll disable this option and let you know.

**Comment by mezz** [comment]
Thanks for the report!
JEI doesn't add that tooltip at all, so it must be from something else.

**Comment by MauveCloud** [comment]
I did a little experimenting.  As far as I can tell, the numeric id and the first internal name (or id string, if you prefer; anyway, it's dark gray) are from vanilla Minecraft (I got it to appear without even having Forge running), but only appear if you use F3+H to show advanced tooltips.  I think the second internal name (in light gray) is from EnderCore, which defaults to showing that "only in debug mode", even though that is now redundant - maybe it's a holdover from older MC versions where vanilla MC didn't provide that information.
